greenpithumb/dht11.py

```diff
@@ -4,9 +4,6 @@
 from Adafruit_DHT import common
```

Comment:
Hmm are we adding it to requirements or adding a greenpithumb/Adafruit_DHT folder like we did with the previous dht11 code? And then add whatever packages they use to ```requirements.txt```?